cranelift/codegen/src/isa/aarch64/lower_inst.rs

```diff
@@ -150,68 +150,27 @@ pub(crate) fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
         Opcode::UaddSat | Opcode::SaddSat | Opcode::UsubSat | Opcode::SsubSat => {
             // We use the scalar SIMD & FP saturating additions and subtractions
             // (SQADD / UQADD / SQSUB / UQSUB), which require scalar FP registers.
```

Comment:
I think I got the below re-factoring correct but this comment may need to change--suggestions?

Reply:
Just remove the comment completely - it's no longer relevant.

Reply:
Thanks!